packages/types/environment.ts

```diff
@@ -27,6 +27,8 @@ export const ZEnvironmentUpdateInput = z.object({
 
 export const ZId = z.string().cuid2();
 
+export const ZUuid = z.string().uuid();
```

Comment:
@Dhruwang This doesn't really make sense in the `environment` types. I was already the case with the commonly used `ZId` before.
Can you please move `ZId` & `ZUuid` into `types/common.ts` and fix the imports?